Features/Context/SubContext/Authentication.php

```diff
@@ -53,7 +53,7 @@ public function goToPlatformUiAndLogIn($username, $password)
      * @Given I am logged in as admin on PlatformUI
      * @Given I am logged in as an Editor in PlatformUI
```

Comment:
Currently it is only possible to login in platformUI with admin user, this is a known issue but I can't seem to find a related Jira issue.
The editor sentence for now is just a place holder using the admin credentials in order for the tests to work